demos/twa-webview-fallback/src/main/java/com/google/browser/examples/twawebviewfallback/MyLauncherActivity.java

```diff
@@ -26,6 +29,13 @@
         return (context, twaIntentBuilder, mProviderPackage, completionCallback) -> {
             Intent intent = new Intent(context, WebViewFallbackActivity.class);
             intent.putExtra(WebViewFallbackActivity.KEY_LAUNCH_URI, twaIntentBuilder.getUri());
+
+            LauncherActivityMetadata metadata = LauncherActivityMetadata.parse(this);
+            intent.putExtra(WebViewFallbackActivity.STATUS_BAR_COLOR,
+                    ContextCompat.getColor(this, metadata.statusBarColorId));
+            intent.putExtra(WebViewFallbackActivity.NAVIGATION_BAR_COLOR,
+                    ContextCompat.getColor(this, metadata.navigationBarColorId));
```

Comment:
What I'd probably do is create a static method called `createLaunchIntent` in WebViewFallbackActivity that takes a URL and the colours (or maybe the URL and the metadata), so this method would become:

    Intent intent = WebViewFallbackActivity.createLaunchIntent(context,
        twaIntentBuilder.getUri(), LauncherActivityMetadata.parse(this));
    startActivity(intent);
    completionCallback.run();

This way, `KEY_LAUNCH_URI`, `NAVIGATION_BAR_COLOR` and `STATUS_BAR_COLOR` can all be private and only WebViewFallbackActivity needs to know exactly how the Intent is formatted.

Reply:
done